core/templates/dev/head/pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.js

```diff
@@ -27,27 +27,91 @@ oppia.directive('subtopicsListTab', [
       controller: [
         '$scope', '$uibModal', 'TopicEditorStateService', 'TopicUpdateService',
         'UndoRedoService', 'EVENT_TOPIC_REINITIALIZED',
-        'EVENT_TOPIC_INITIALIZED',
+        'EVENT_TOPIC_INITIALIZED', 'EVENT_SUBTOPIC_PAGE_LOADED',
         function(
             $scope, $uibModal, TopicEditorStateService, TopicUpdateService,
             UndoRedoService, EVENT_TOPIC_REINITIALIZED,
-            EVENT_TOPIC_INITIALIZED) {
+            EVENT_TOPIC_INITIALIZED, EVENT_SUBTOPIC_PAGE_LOADED) {
+          $scope.subtopicDisplayed = false;
```

Comment:
Add one-line comment above this explaining what it means.

Reply:
Done.